### ISSUE TITLE
Improve representation of hosting institutions

### ISSUE THREAD

**Issue by ahahn-gbif** [gbif/portal-feedback#1382]
The role of data host was introduced to allow proper credit to both the owner ("publisher") and the institution supporting publication ("host"). In contrast to those publishers who host their own datasets, though, publishers who (almost exclusively) act as hosts for the datasets of other registered publishers still have relatively limited visibility. The publisher page exists, but only lists a very few numbers (dataset count with filter link, publisher count, countries) and appears largely empty. Example: https://www.gbif.org/publisher/a86e9e36-12ec-49a4-a94c-c0c981fffb71. 

To give appropriate credit to institutions that act as hosts, and to motivate and support hosts in their important function, the following improvements to the page are considered:

1. an occurrence map including occurrences from hosted datasets (hostingKey)
2. a list of the publishers supported by the host [a list of datasets already exists, and should remain]
3. an occurrence search by hostingKey, and consequently 
4. counts and metrics for hosted data (min. starting point: occurrence count)

NB: some hosts have a mixed model, publishing own datasets as well as hosting datasets for other institutions. Both types of data need to be included.

**Comment by myrmoteras** [comment]
just to make sure:
"Publisher" = journal article publisher
"host"= data publisher = data publisher sensu GBIF.
![image](https://user-images.githubusercontent.com/4609956/41039254-2aa0fd18-6999-11e8-8339-4c81dfd50d60.png)

**Comment by myrmoteras** [comment]
is there a possibility to to add "counts and or lists of the taxon submitted by the host?"

**Comment by ahahn-gbif** [comment]
Sorry, that was confusing - I added a few more words. No, not quite:
"Publisher" = publishing organization sensu GBIF, i.d. a registered organization, endorsed by a GBIF Participant node, that can publish dataset through the network
"Host" = likewise a GBIF-registered and endorsed organization, hosting datasets for own or other organizations.
For most registered organizations, the two roles of "publisher" and "host" coincide - they host their own datasets. In that case, the role of the host is not specially emphasized. In cases where the publisher/owner of the dataset and the host are different organizations, the hosting role of the organization is made explicit.

**Comment by ahahn-gbif** [comment]
Yes, we should include checklist datasets as well. Thanks for reminding!

**Comment by myrmoteras** [comment]
@ahahn-gbif what will be the implication for the upload of new DWCA for the host, i.e. Guido?

**Comment by ahahn-gbif** [comment]
- the GBIF publisher entry needs to exist, or be created (https://www.gbif.org/become-a-publisher; there is an endorsement workflow attached, but in your case, the loop is quite short)
- the dataset is tied against two instead of one institution / publisher entries: the host (Plazi), and the owner (journal)

Re-linking the existing entries is likely best done from here, but we would need the GBIF publishers to exist before that is possible.
